includes/Rewrites.php

```diff
@@ -329,4 +330,25 @@ public function store_query_filter( $query ) {
             $query->set( 'orderby', 'post_date ID' );
         }
     }
+
+    /**
+     * Store listing page make order by random
+     *
+     * @since DOKAN_LITE_SINCE
+     *
+     * @param obj $class
+     *
+     * @return obj $class
+     */
+    public function random_store_query( $class ) {
+        if ( ! dokan_is_store_listing() ) {
+            return $class;
+        }
+
+        if ( 'random' === $class->query_vars['orderby'] ) {
+            $class->query_orderby = str_replace( 'user_login', 'RAND()', $class->query_orderby );
```

Comment:
using RAND() will give you random results each time, this will mess up pagination, you can't use it like that, kindly remove this code.